app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(
+                "You need to be vaccinated to visit the cafe!"
+            )
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date and expiration_date < date.today():
+            raise OutdatedVaccineError(
+                "The vaccination has expired!"

```

Comment:
Code Style: Again, include the name of the visitor in the error message. For example, `f'{visitor['name']}, your vaccination has expired!'`.